components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway.v2/src/gen/java/org/wso2/carbon/apimgt/rest/api/gateway/v2/SequenceApiService.java

```diff
@@ -1,16 +1,16 @@
-package org.wso2.carbon.apimgt.rest.api.gateway.v1;
+package org.wso2.carbon.apimgt.rest.api.gateway.v2;
 
-import org.wso2.carbon.apimgt.rest.api.gateway.v1.*;
-import org.wso2.carbon.apimgt.rest.api.gateway.v1.dto.*;
+import org.wso2.carbon.apimgt.rest.api.gateway.v2.*;
```

Comment:
shall we remove org.wso2.carbon.apimgt.rest.api.gateway.v2 and keep only org.wso2.carbon.apimgt.rest.api.gateway. This is troublesome to update during releases